### ISSUE TITLE
<Touchable accessibilityRole="link" href /> does not work with keyboard

### ISSUE THREAD

**Issue by MoOx** [necolas/react-native-web#643]
**Do you want to request a *feature* or report a *bug*?**

Bug

**What is the current behavior?**

If you navigate with the keyboard and use "enter", you get the opacity change, but not the normal link behavior.

```js
<TouchableOpacity
    accessibilityRole="link"
    href={href}
/>
```

You can compare on https://tart-attempt.glitch.me one link with TouchableOpacity (the first one) and one normal link.
Both works with the mouse, but not with the keyboard.

See code used for the demo https://glitch.com/edit/#!/tart-attempt?path=src/App/index.js:21:36

**What is the expected behavior?**

Using "enter" to click on a link should work with Touchable.
I suspect this line to be responsible https://github.com/necolas/react-native-web/blob/master/src/components/Touchable/Touchable.js#L822

**Environment (include versions). Did this work in previous versions?**

* OS: macOS
* Device: -
* Browser: all
* React Native for Web (version): 0.0.125
* React (version): 0.15.4
